### PR TITLE
ft: ZENKO-1560 add getIngestionBuckets to MongoClientInterface

### PR DESCRIPTION
**Pull request by philipyoo**
Add `getIngestionBuckets` to MongoClientInterface and expose through MetadataWrapper

### CODE REVIEW

**Review by alexanderchan-scality — APPROVED** [review]

**Review by JianqinWang — APPROVED** [review]

File: lib/storage/metadata/MetadataWrapper.js [hunk 1]
```diff
@@ -344,6 +344,10 @@ class MetadataWrapper {
     notifyBucketChange(cb) {
         bucketNotificationHook = cb;
     }
+
```
Comment (rahulreddy) [inline lib/storage/metadata/MetadataWrapper.js]: I would rather have the method you intended to use here than exporting the client. It defeats the purpose of having a wrapper.

**Review by rahulreddy — CHANGES_REQUESTED** [review]
I would rather have the method you intended to use here than exporting the client. It defeats the purpose of having a wrapper.

**Comment by philipyoo** [comment]
@rahulreddy 
The method is very specific to only ingestion within backbeat and the MetadataWrapper can create different internal clients. I don't think it would fit well on the wrapper either

The alternative was to have a second mongodb instance within backbeats lib/management; one that continues to use the existing MetadataWrapper, and the other that extends the MongoClientInterface (like how we were using it before). Basically 2 separate connections to the same mongo db

**Review by rahulreddy — APPROVED** [review]

**Comment by philipyoo** [comment]
@bert-e approve

**Pull request merged**